### PR TITLE
Refactor handling reward tokens

### PR DESCRIPTION
**Pull request by wohlrich**
Track total number of tokens minted to use with calculating per token rewards.
Remove rewardTokenBalance and hard-coded totalRewardToken.

The contract tracks the amount of rewards per token claimed per account.
Accounts automatically claim any unclaimed reward tokens before any transfers 
into or out of the account occurs, so the balances should be up-to-date during any transaction.

### CODE REVIEW

File: contracts/sktl20.sol [hunk 1]
```diff
@@ -8,21 +8,20 @@ import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
 contract Sktl20 is ERC20 {
     uint256 public constant scaling = 1000000000000000000; // 10^18
-    uint256 public constant totalRewardToken = 1000000000000000000000000000; // about 10B, and should be fixed
+    uint256 public totalTokens = 0;
```
Comment (ChromeHearts) [inline contracts/sktl20.sol:11]: No need to keep track of totalToken.  Use totalSupply()

File: contracts/sktl20.sol [hunk 2]
```diff
@@ -31,20 +30,25 @@ contract Sktl20 is ERC20 {
         virtual
         returns (uint256)
     {
+	// Calculate scaled value owed per token for account
         uint256 scaledOwed = scaledRewardPerToken -
             scaledRewardCreditedTo[account];
-        return (rewardTokenBalance[account] * scaledOwed) / scaling;
+        // Return unclaimed portion of total rewards for account
+        // TODO - Does balanceOf return tokens or wei?  What aboouot reward_balance?
```
Comment (ChromeHearts) [inline contracts/sktl20.sol:37]: Remove comments, everything should be in Wei

**Review by ChromeHearts — CHANGES_REQUESTED** [review]

**Comment by ChromeHearts** [comment]
@wohlrich I understand what's the problem, **sktl.sol** is actual the latest contract.   Sorry, the sktl20.sol is deprecated.

**Comment by wohlrich** [comment]
closing PR to deprecated file.

**Pull request closed without merging**